pages/index.tsx

```diff
@@ -11,7 +11,7 @@ import {
   ProgressLoader,
   VisitedItemsList,
 } from "../components";
-import { FavoritePin } from "../components/FavoritePin";
+import { FavoritePin } from "../components/FavoritePin/FavoritePin";
```

Comment:
For consistency export FavoritePin in `components/index.ts`, so it can be imported here in the list 

```js
import {
  ExternalLink,
  FavoritePin,
  Footer,
  Header,
  HeadMeta,
  NewFeedForm,
  NewItemsList,
  ProgressLoader,
  VisitedItemsList,
} from "../components";
```